src/taskConstructor.js

```diff
@@ -0,0 +1,9 @@
+class Task {
+  constructor(value, index) {
+    this.description = value;
+    this.completed = false;
+    this.index = index;
+  }
```

Comment:
- [ ] As requested in the project requirement, make sure that once you click on the checkbox, the methods related to the status updates are updated in the local storage(means the completed should be true in your local storage once you check the checkbox)